The Getting started guide missing a prepocessor annotation

<!-- STOP

* If this is an issue with some sort of runtime mechanics, it probably belongs in https://sigs.k8s.io/controller-runtime instead
* If this is an issue with CRD generation or webhook config generation, it probably belongs in sigs.k8s.io/controller-tools instead
* If this is an issue with scaffolding, or is definitely a cross repository effort, it probably belongs here.

-->

<!-- 

Hiya!  Welcome to KubeBuilder!  For a smooth issue process, try to answer the following questions.
Don't worry if they're not all applicable; just try to include what you can :-)

If you need to include code snippets or logs, please put them in fenced code
blocks.  If they're super-long, please use the details tag like
<details><summary>super-long log</summary> lots of stuff </details>

-->

<!-- What broke (please include exact error messages if you can) -->
# In the documentation there is an example that probably misses an annotation
<!-- What did you expect to happen?  What do you think went wrong? -->
I expected the snippet to work, but the DeepCopyObject is not created so the `make run` exits with this error
```

api/v1/guestbook_types.go:76:25: cannot use &Guestbook literal (type *Guestbook) as type runtime.Object in argument to SchemeBuilder.Register:
	*Guestbook does not implement runtime.Object (missing DeepCopyObject method)

```
<!-- If you don't know what broke, but have a minimal reproducer, that would be super-helpful helpful! -->
It can be replicated by following the gettting started guide including the optional part
<!--

What versions of software are you using?  Specifically, the following are often useful:

* go version
* kubebuilder version (`kubebuilder version`) and scaffolding version (check your `PROJECT` file)
* controller-runtime version (check your `go.mod` file)
* controller-tools version
* Kubernetes & kubectl versions (just run `kubectl version` against your API server)

-->
- go version go1.14 linux/amd64
- kubebuilder version: 
  - KubeBuilderVersion:"2.3.1", 
  - KubernetesVendor:"1.16.4", 
  - GitCommit:"8b53abeb4280186e494b726edf8f54ca7aa64a49"
- controller-tools version: ?
- kubernetes:
  - kubectl: v1.17.0
  - server: v1.16.8-eks-e16311
<!-- If this is actually about documentation, add `/kind documentation` below -->

/kind documentation

--- a/docs/book/src/quick-start.md
+++ b/docs/book/src/quick-start.md
@@ -131,6 +131,11 @@ type GuestbookStatus struct {
 	Standby []string `json:"standby"`
 }
 
+// +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster
+
+// Guestbook is the Schema for the guestbooks API
 type Guestbook struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`

